lib/generators/react_on_rails/templates/base/base/babel.config.js

```diff
@@ -0,0 +1,103 @@
+module.exports = function (api) {
```

Comment:
This is going to be hard to maintain.

Any thoughts on how we can rely more on rails/webpacker's defaults and then maybe modify those?